bundle/manifests/operator.istio.io_istios.yaml

```diff
@@ -55,6 +55,10 @@ spec:
                 description: The built-in installation configuration profile to use.
                   When this field is left empty, the 'default' profile will be used.
                 type: string
+              rawValues:
+                description: RawValues defines the values to be passed to the Helm
+                  chart when installing Istio and that do not need a CRD validation.
```

Comment:
Maybe explain that the values here don't represent a stable API or something to that effect. And I don't think you should say "CRD validation"... Instead, just say something like "these values aren't validated".